core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -353,7 +354,7 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
                 traceWithParent("getUserComputePermissions", s1)(_ =>
                   getUserComputePermissions(workspaceContext.workspaceIdAsUUID.toString,
                                             accessLevel,
-                                            Some(wsmContext.getCloudPlatform)
+                                            wsmContext.getCloudPlatform
```

Comment:
If this ends up being None`, `getUserComputePermissions` will default to GCP permission model (Azure is slightly more lenient):

```
  private def getUserComputePermissions(workspaceId: String,
                                        userAccessLevel: WorkspaceAccessLevel,
                                        cloudPlatform: Option[WorkspaceCloudPlatform]
  ): Future[Boolean] =
    cloudPlatform match {
      case Some(WorkspaceCloudPlatform.Azure) =>
        Future.successful(userAccessLevel >= WorkspaceAccessLevels.Write)
      case _ if userAccessLevel >= WorkspaceAccessLevels.Owner =>
        Future.successful(true)
      case default =>
        samDAO.userHasAction(SamResourceTypeNames.workspace, workspaceId, SamWorkspaceActions.compute, ctx)
    }